mwdb/web/src/components/ShowObject/common/CommentBox.jsx

```diff
@@ -0,0 +1,96 @@
+import { useState, useContext, useEffect, useCallback } from "react";
```

Comment:
I see it's still untyped jsx. Do you want to make types for it in next PR or something wasn't committed?

Reply:
i simply forget to rewrite this files ;p